static/test/javascripts/spec/common/commercial/membership-engagement-banner.spec.js

```diff
@@ -69,13 +72,15 @@ define([
 
         function expectMessageNotToBeShown(done) {
             membershipMessages.init().then(function () {
+                mediator.emit('modules:onwards:breaking-news:ready', false);
                 var message = document.querySelector('.js-site-message');
                 expect(message).toBeNull();
             }).then(done);
         }
 
         function expectMessageNotToBeVisible(done) {
             membershipMessages.init().then(function () {
+                mediator.emit('modules:onwards:breaking-news:ready', false);
```

Comment:
Maybe we should have test(s) where `true` is emitted from the breaking news component?